pulsar-client-cpp/lib/ConsumerImpl.cc

```diff
@@ -483,13 +484,13 @@ inline proto::CommandSubscribe_SubType ConsumerImpl::getSubType() {
     ConsumerType type = config_.getConsumerType();
     switch (type) {
         case ConsumerExclusive:
-            return proto::CommandSubscribe_SubType_Exclusive;
```

Comment:
both mean the same - changed since underscores were ugly 

  static const SubType Exclusive = CommandSubscribe_SubType_Exclusive;
  static const SubType Shared = CommandSubscribe_SubType_Shared;
  static const SubType Failover = CommandSubscribe_SubType_Failover;